components/button/Button.styles.ts

```diff
@@ -14,4 +14,8 @@ export const Wrapper = styled.button`
   font-weight: 700;
   text-align: center;
   cursor: pointer;
+  :disabled {
+    background-color: #7fbce9;
+    cursor: default;
```

Comment:
Why "cursor: pointer"? If an element is disabled, theres a "disabled" cursor automatically.

Reply:
The cursor pointer needs to be just like the Reddit site without it there will be a different CSS style.